File: PerfectNumber/PerfectNumber.cs
// WAP to check that given number is Perfect or not

using System;
class PerfectNumber
{
    static void Main()
    {
        int n, i, sum = 0;
        Console.Write("Enter Number: ");
        n = int.Parse(Console.ReadLine());

        for (i = 1; i <= n / 2; i++)
        {
            if (n % i == 0)
            {
                sum += i;
            }
        }
        if (sum == n)
        {
            Console.WriteLine($"{n} is a Perfect Number");
        }
        else
        {
            Console.WriteLine($"{n} is not a Perfect Number");
        }
    }
}

File: PerfectNumber/PerfectNumber.py
# WAP that check given number is Perfect or not.

n = int(input("Enter number: "))
sum = 0
for i in range(1, n // 2 + 1):
    if n % i == 0:
        sum += i
if sum == n:
    print(f"{n} is a Perfect Number")
else:
    print(f"{n} is not a Perfect Number")
